go.mod

```diff
@@ -1,15 +1,3 @@
-module github.com/trussworks/terraform-module-template
+module terraform-aws-ou-scp
```

Comment:
Can this file just be deleted?

Reply:
Think so... there aren't terratests to support with this.